dispersy.py

```diff
@@ -1890,21 +1975,21 @@ def select(sql, bindings):
                 meta_identity = community.get_meta_message(u"dispersy-identity")
 
                 try:
-                    member_id, = self._database.execute(u"SELECT id FROM member WHERE mid = ?", (buffer(community.my_member.mid),)).next()
-                except StopIteration:
+                    member_id, = yield self._database.stormdb.fetchone(u"SELECT id FROM member WHERE mid = ?", (buffer(community.my_member.mid),))
+                except TypeError:
                     raise ValueError("unable to find the public key for my member")
 
                 if not member_id == community.my_member.database_id:
                     raise ValueError("my member's database id is invalid", member_id, community.my_member.database_id)
 
                 try:
-                    self._database.execute(u"SELECT 1 FROM member WHERE id = ? AND private_key IS NOT NULL", (member_id,)).next()
-                except StopIteration:
+                    _, = yield self._database.stormdb.fetchone(u"SELECT 1 FROM member WHERE id = ? AND private_key IS NOT NULL", (member_id,))
```

Comment:
If you are ignoring the return value, why not just delete the left hand of the assignment?